smartvio/syzygy.c

```diff
@@ -155,6 +156,14 @@ szgSolveSmartVIOGroup(szgSmartVIOPort *ports, int group_mask)
 				continue;
 			}
 
+			// Check the minimum support version of the peripheral in the target port
```

Comment:
I was unsure about placing this check in our current solver code or adding a separate solver function. I landed on adding it to our current solver since we already iterate over all ports here and doing so a second time just to check against the required version seems excessive, but I could be convinced otherwise.